recipe/meta.yaml

```diff
@@ -15,9 +15,10 @@ source:
       - hdf5-1.12.patch
       - fix-xdmf.patch
       - python-cmake-args.patch
+      - boost-1.86.patch
```

Comment:
feel free to add yourself to the recipe-maintainers down at the bottom of this file

Reply:
Okay, I did so with the next commit.